OloEngine/src/OloEngine/Scene/Scene.cpp

```
@@ -1303,23 +1307,394 @@ namespace OloEngine
+            {
+                shadowMap.ComputeCSMCascades(
+                    directionalLightDir,
+                    camera.GetViewMatrix(),
+                    camera.GetProjection(),
+                    camera.GetNearClip(),
+                    camera.GetFarClip());
+            }
+
+            // Spot light shadows
+            {
+                u32 spotIdx = 0;
```

Clang (14.0.6)

[warning] variable 'spotIdx' is not initialized

(cppcoreguidelines-init-variables)